src/sentry/new_migrations/migrations.py

```diff
@@ -7,7 +7,7 @@ class CheckedMigration(Migration):
     won't cause production issues during deploy.
     """
 
-    checked = True
```

Comment:
Could you revert this? You only need to set `checked=False` in your specific migration, not on the base class here.

Reply:
Yup for sure, there might be a few things that aren't meant to be here but I didn't remove it as I didn't plan to merge this pr